### DISCUSSION TITLE
Date cut-off in Group by helper

### DISCUSSION THREAD

**Discussion by phoob** [microsoft-search/pnp-modern-search#1389] [Q&A]
I have a date column for the news articles I want to display in the search results. The dates can be in the past or in the future. I have tried using the {{#group by}} helper to separate those articles with past dates from ones in the future, but this does not seem to be possible.

I would like for it to be possible to use an expression in the "by" parameter, ie 
```
{{#group by=(moment RefinableDate00 isAfter=today)}}
```

Thanks for considering this!

**Comment by FranckyC** [comment]
Hi @phoob, the `group` helper doesn't work in the current release but is fixed in the `develop` branch. See https://github.com/microsoft-search/pnp-modern-search/issues/1352

**Comment by phoob** [comment]
I have tried using develop branch now, but I don't get it to work. 

This "works":
```
{{#group data.items by="RefinableDate01"}}
  {{value}}
{{/group}}
```
– I get a list of the different dates.

But I only want two groups: one where "RefinableDate01" is before or equal to today, and one where it is after today. 

So I tried this:

```
{{#group data.items by="(moment RefinableDate01 isAfter=(getDate (moment) 'YYYY-MM-DD'))"}}
  {{value}}
{{/group}
}
```

But that does not display anything... 

Also this does not work:

```
{{#group data.items by="(moment RefinableDate01 format='YYYY-MM-DD')"}}
  {{value}}
{{/group}
}
```

and I also tried this – doesn't work:

```
{{#group data.items by=(moment RefinableDate01 format="YYYY-MM-DD")}}
  {{value}}
{{/group}
}
```

Side note: the following code works:

```
{{#if (moment (slot item @root.slots.StartDate) isAfter=(getDate (moment) "YYYY-MM-DD"))}}
  <span class="text-color--black">Planned from {{moment (slot item @root.slots.StartDate) format="L"}}: </span>
{{/if}} 
```
